src/app/services/sms/sms.factory.ts

```diff
@@ -0,0 +1,65 @@
+import Twilio from 'twilio'
+
+import FeatureManager, {
+  FeatureNames,
+  RegisteredFeature,
+} from '../../../config/feature-manager'
+
+import { sendAdminContactOtp, sendVerificationOtp } from './sms.service'
+import { TwilioConfig } from './sms.types'
+
+interface ISmsFactory {
```

Comment:
move these to the types folder?

Reply:
I use `typeof` to get the types leh, I scared circular dependency :X

Reply:
Add a comment for future developers?